awxkit/awxkit/api/pages/credentials.py

```diff
@@ -94,6 +94,7 @@ def config_cred_from_kind(kind):
     'machine': 'ssh',
     'vault': 'vault',
     'vmware vcenter': 'vmware',
+    'gpg public key': 'gpg_public_key',
```

Comment:
I think we need to add `gpg_public_key` as an input field above too (line 28)

Reply:
haha you were right haha thanks